spec/acts_as_taggable_on/dirty_spec.rb

```diff
@@ -20,7 +20,11 @@
       it 'should show changes of freshly initialized dirty object' do
         taggable = TaggableModel.find(@taggable.id)
         taggable.tag_list = 'one'
-        expect(taggable.changes).to eq({'tag_list' => [['awesome', 'epic'], ['one']]})
+        if Rails.version >= "6.0.0.beta1"
```

Comment:
I don't think this is the right call; I have tried fixing this but I'd suggest starting this it block with 

`pending if Rails.version >= "6.0.0.beta1"`

and remove the conditional. Here you're silently allowing a possibly breaking change to simply pass the tests